src/subscription_manager/repofile.py

```diff
@@ -585,19 +589,21 @@ def read_zypp_conf(self):
             self.gpgcheck = zypp_cfg.getboolean('rhsm-plugin', 'gpgcheck')
         if zypp_cfg.has_option('rhsm-plugin', 'autorefresh'):
             self.autorefresh = zypp_cfg.getboolean('rhsm-plugin', 'autorefresh')
+        if zypp_cfg.has_option('rhsm-plugin', 'gpgkey-ssl-verify'):
+            self.gpgkey_ssl_verify = zypp_cfg.getboolean('rhsm-plugin', 'gpgkey-ssl-verify')
```

Comment:
I believe that `gpgkey-ssl-verify` should not be boolean, but the type of this option should be string with following valid values: `yes`, `no`, `peer` and `host`.